<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>675</width>
    <height>585</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>任务管理器</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>3</number>
     </property>
     <widget class="QWidget" name="tab_1">
      <attribute name="title">
       <string>系统</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0" colspan="3">
        <widget class="QWidget" name="widget" native="true">
         <widget class="QWidget" name="widget_3" native="true">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>260</y>
            <width>541</width>
            <height>171</height>
           </rect>
          </property>
          <widget class="QLabel" name="ram_Label">
           <property name="geometry">
            <rect>
             <x>260</x>
             <y>70</y>
             <width>231</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_9">
           <property name="geometry">
            <rect>
             <x>120</x>
             <y>30</y>
             <width>61</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>处理器</string>
           </property>
          </widget>
          <widget class="QLabel" name="cpu_Label">
           <property name="geometry">
            <rect>
             <x>260</x>
             <y>30</y>
             <width>301</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_11">
           <property name="geometry">
            <rect>
             <x>120</x>
             <y>110</y>
             <width>61</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>硬盘容量</string>
           </property>
          </widget>
          <widget class="QLabel" name="harddisk_Label">
           <property name="geometry">
            <rect>
             <x>260</x>
             <y>120</y>
             <width>221</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_4">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>20</y>
             <width>61</width>
             <height>32</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>20</pointsize>
            </font>
           </property>
           <property name="text">
            <string>硬件</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_10">
           <property name="geometry">
            <rect>
             <x>120</x>
             <y>70</y>
             <width>61</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>内存容量</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="widget_2" native="true">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>10</y>
            <width>541</width>
            <height>71</height>
           </rect>
          </property>
          <widget class="QLabel" name="label_8">
           <property name="geometry">
            <rect>
             <x>120</x>
             <y>30</y>
             <width>61</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>用户名</string>
           </property>
          </widget>
          <widget class="QLabel" name="username_Label">
           <property name="geometry">
            <rect>
             <x>260</x>
             <y>30</y>
             <width>111</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
          <widget class="QLabel" name="label">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>20</y>
             <width>61</width>
             <height>32</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>20</pointsize>
            </font>
           </property>
           <property name="text">
            <string>用户</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="widget_4" native="true">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>90</y>
            <width>541</width>
            <height>161</height>
           </rect>
          </property>
          <widget class="QLabel" name="label_7">
           <property name="geometry">
            <rect>
             <x>120</x>
             <y>110</y>
             <width>61</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>窗口系统</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_5">
           <property name="geometry">
            <rect>
             <x>120</x>
             <y>30</y>
             <width>61</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>版本</string>
           </property>
          </widget>
          <widget class="QLabel" name="kernel_Label">
           <property name="geometry">
            <rect>
             <x>260</x>
             <y>70</y>
             <width>231</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_3">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>20</y>
             <width>61</width>
             <height>32</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>20</pointsize>
            </font>
           </property>
           <property name="text">
            <string>系统</string>
           </property>
          </widget>
          <widget class="QLabel" name="version_Label">
           <property name="geometry">
            <rect>
             <x>260</x>
             <y>30</y>
             <width>181</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
          <widget class="QLabel" name="windowsys_Label">
           <property name="geometry">
            <rect>
             <x>260</x>
             <y>110</y>
             <width>171</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_6">
           <property name="geometry">
            <rect>
             <x>120</x>
             <y>70</y>
             <width>61</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>内核</string>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
       <item row="1" column="0">
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>450</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="halt_Button">
         <property name="text">
          <string>关机</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="reboot_Button">
         <property name="text">
          <string>重启</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>进程</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="label_24">
         <property name="text">
          <string>前一分钟，五分钟，十五分钟的平局负载为</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QLabel" name="cpuaverage_Label">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="QTreeWidget" name="process_TreeWidgt">
         <attribute name="headerShowSortIndicator" stdset="0">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>进程名</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>pid</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>%CPU</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>内存</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>优先级</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>队列</string>
          </property>
         </column>
         <item>
          <property name="text">
           <string>新建项目</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="text">
           <string/>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="1">
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>288</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="pkill_Button">
         <property name="text">
          <string>结束进程</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>网络</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QTreeWidget" name="network_TreeWidgt">
         <column>
          <property name="text">
           <string>进程</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>pid</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>上传速率</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>下载速率</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>上传量</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>下载量</string>
          </property>
         </column>
         <item>
          <property name="text">
           <string>新建项目</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>资源</string>
      </attribute>
      <widget class="QFrame" name="frame">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>40</y>
         <width>571</width>
         <height>66</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_8">
        <item row="0" column="0">
         <widget class="QLabel" name="label_12">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>CPU</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QProgressBar" name="cpu_ProgressBar">
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QFrame" name="frame_3">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>150</y>
         <width>571</width>
         <height>140</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="0" column="0">
         <widget class="QLabel" name="label_18">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>交换空间</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QProgressBar" name="swap_ProgressBar">
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_21">
            <property name="text">
             <string>总量</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_SWAP_Total">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_23">
            <property name="text">
             <string>已用</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_SWAP_Used">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_20">
            <property name="text">
             <string>可用</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_SWAP_Left">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QFrame" name="frame_2">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>320</y>
         <width>571</width>
         <height>140</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="2" column="0">
         <widget class="QLabel" name="label_17">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>内存</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QProgressBar" name="ram_ProgressBar">
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_27">
            <property name="text">
             <string>总量</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_RAM_Total">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_29">
            <property name="text">
             <string>已用</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_RAM_Used">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_31">
            <property name="text">
             <string>可用</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_RAM_Left">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>文件系统</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QTreeWidget" name="filesys_TreeWidgt">
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <attribute name="headerShowSortIndicator" stdset="0">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>磁盘</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>类型</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>容量</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>已用</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>剩余</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>路径</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_6">
      <attribute name="title">
       <string>说明</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
